packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -375,143 +373,66 @@ async function uploadConsolidatedJsonAndReturnUrl({
  */
 export async function getConsolidatedPatientData({
```

Comment:
Should we update the tsdoc for this function to explain it uses the lambda

Reply:
done